src/service/server.js

```diff
@@ -48,20 +51,24 @@ function listenCallback(tls) {
   log.info('Workspace: ' + (tls ? 'https' : 'http') + `://localhost:${config.port}/ui/wwe/index.html`);
 }
 
-// try to host the server in https
-try {
-  // load the certificates
-  const certs = {
-    key: fs.readFileSync(path.join(__dirname, '../../data/certificates/localhost.key.pem')),
-    cert: fs.readFileSync(path.join(__dirname, '../../data/certificates/localhost.cert.pem'))
-  };
-  // listen
-  https.createServer(certs, app).listen(config.port, () => listenCallback(true));
-}
-catch (e) {
-  // if the certificates were not found, start in http
-  log.info('Failed to start the HTTPS server, starting with HTTP. \n' + e);
+if (isTesting || config.https) {
```

Comment:
We now have a config.https property, if we want to run the simulator in http mode.
If we are testing, it always runs in https.